starter_code/bin/www

```diff
@@ -41,6 +41,10 @@ server.on('error', error => {
   }
 });
 
-server.listen(process.env.PORT, () => {
+// server.listen(process.env.PORT || 3000, () => {
+//   console.log(`Listening on http://localhost:${process.env.PORT}`);
+// });
+
+server.listen(3000, () => {
```

Comment:
If you want to protect you're PORT make sure to do this `process.env.PORT || 3000` or `process.env.PORT` and add `PORT=3000` in the `.env`file